runtime/src/debug.js

```diff
@@ -0,0 +1,128 @@
+import indentString from "indent-string";
+import { isVnode } from './equality';
+import { Variant } from './variant';
+import { Name } from './symbols';
```

Comment:
Use either single or double quotes consistently.
Even better would be to configure ESLint/Prettier to do it automatically.

ditto elsewhere

Reply:
We have Prettier set up https://github.com/mint-lang/mint/blob/master/runtime/Makefile#L21 but at this time it would produce many unrelated changes, so it would be better to run it idependetly in an other commit.